qiskit_nature/mappers/second_quantization/bksf.py

```diff
@@ -0,0 +1,583 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Bravyi-Kitaev Super Fast(BKSF) Mapper."""
+
+from enum import Enum
+from typing import List, Tuple
+import numpy as np
+
+from qiskit.opflow import PauliSumOp
+from qiskit.quantum_info.operators import Pauli, SparsePauliOp
+from qiskit_nature.operators.second_quantization import FermionicOp
+from .fermionic_mapper import FermionicMapper
+
+
+class BravyiKitaevSuperFastMapper(FermionicMapper):
+    """The Bravyi-Kitaev super fast fermion-to-qubit mapping.
+
+    This implementation follows closely Reference [1].
+
+       References:
+           [1]: Bravyi-Kitaev Superfast simulation of electronic structure on a quantum computer
```

Comment:
This appears to be the intended way of typesetting `Superfast`. Maybe we should stick to that throughout the docstrings? E.g. a few lines higher we instead write `super fast`...

Reply:
I did a quick search and found examples of BKSF referred to as "superfast", "super fast" and "super-fast". The author of that paper (Whitfield) uses "super-fast" on his webpage and in a grant application.
I prefer "super-fast" since it follows standard hyphenation rules and coining the word "superfast" seems like a bit of overreach, even in the physics world.

What we have now, "super fast", is the worst of the three. For the moment, I am changing it to "super-fast".